src/gui/mainwindow.cpp

```diff
@@ -1107,6 +1108,8 @@ void MainWindow::loadPreferences(bool configure_session)
         toolBar->setVisible(false);
     }
 
+    QMainWindow::statusBar()->setVisible(pref->isStatusbarDisplayed());
```

Comment:
However, you'll need to move the status_bar creation before the call of loadPreferences() in the constructor.